MaterialDesignThemes.Wpf/CustomBaseColorTheme.cs

```diff
@@ -0,0 +1,156 @@
+        public Color MaterialDesignCheckBoxOff  { get; set; } = (Color) ColorConverter.ConvertFromString("#89000000");
+        public Color MaterialDesignCheckBoxDisabled  { get; set; } = (Color) ColorConverter.ConvertFromString("#FFBDBDBD");
+        public Color MaterialDesignTextBoxBorder  { get; set; } = (Color) ColorConverter.ConvertFromString("#89000000");
+        public Color MaterialDesignDivider  { get; set; } = (Color) ColorConverter.ConvertFromString("#1F000000");
+        public Color MaterialDesignSelection  { get; set; } = (Color) ColorConverter.ConvertFromString("#FFDEDEDE");
+        public Color MaterialDesignToolForeground  { get; set; } = (Color) ColorConverter.ConvertFromString("#FF616161");
+        public Color MaterialDesignToolBackground  { get; set; } = (Color) ColorConverter.ConvertFromString("#FFE0E0E0");
+        public Color MaterialDesignFlatButtonClick  { get; set; } = (Color) ColorConverter.ConvertFromString("#FFDEDEDE");
+        public Color MaterialDesignFlatButtonRipple  { get; set; } = (Color) ColorConverter.ConvertFromString("#FFB6B6B6");
+        public Color MaterialDesignToolTipBackground  { get; set; } = (Color) ColorConverter.ConvertFromString("#757575");
+        public Color MaterialDesignChipBackground  { get; set; } = (Color) ColorConverter.ConvertFromString("#12000000");
+        public Color MaterialDesignSnackbarBackground  { get; set; } = (Color) ColorConverter.ConvertFromString("#FF323232");
+        public Color MaterialDesignSnackbarMouseOver  { get; set; } = (Color) ColorConverter.ConvertFromString("#FF464642");
+        public Color MaterialDesignSnackbarRipple  { get; set; } = (Color) ColorConverter.ConvertFromString("#FFB6B6B6");
+        public Color MaterialDesignTextFieldBoxBackground  { get; set; } = (Color) ColorConverter.ConvertFromString("#0F000000");
+        public Color MaterialDesignTextFieldBoxHoverBackground  { get; set; } = (Color) ColorConverter.ConvertFromString("#14000000");
+        public Color MaterialDesignTextFieldBoxDisabledBackground  { get; set; } = (Color) ColorConverter.ConvertFromString("#08000000");
+        public Color MaterialDesignTextAreaBorder  { get; set; } = (Color) ColorConverter.ConvertFromString("#BC000000");
+        public Color MaterialDesignTextAreaInactiveBorder  { get; set; } = (Color) ColorConverter.ConvertFromString("#0F000000");
+        public Color MaterialDesignDataGridRowHoverBackground  { get; set; } = (Color) ColorConverter.ConvertFromString("#0A000000");
+
+        public void SetTheme()
+        {
+            if (PrimaryColor is Color primaryColor &&
+                SecondaryColor is Color secondaryColor)
+            {
+                var theme = Theme.Create(this, primaryColor, secondaryColor);
+                ApplyTheme(theme);
+            }
+        }
+
+        protected virtual void ApplyTheme(ITheme theme)
+        {
+            this.SetTheme(theme);
+        }
+
+        static CustomBaseColorTheme FromDark()
```

Comment:
Method appears to be un-used.

Reply:
The main ideia is about to provide a shortcut to create a theme, but I've made a mistake because I already wrote the same method on ThemeCreator app (DempApp\Domain\BrushColor.cs).

I'll remove these. Thanks